### PR TITLE
feat: update dataset node 

### LINKED ISSUE #1084: Dataset detail pane

### PR DESCRIPTION
**Pull request by echl**
image

image

Resolves #1084

### CODE REVIEW

File: packages/client/hmi-client/src/components/workflow/dataset-operation.ts [hunk 1]
```diff
@@ -6,5 +6,5 @@ export const DatasetOperation: Operation = {
 	isRunnable: true,
 	inputs: [],
 	outputs: [{ type: 'dataset' }],
-	action: () => {}
+	action: () => { }
```
Comment (echl) [inline packages/client/hmi-client/src/components/workflow/dataset-operation.ts:9]: This happened automatically on merge

File: packages/client/hmi-client/src/components/workflow/tera-simulation-workflow.vue [hunk 2]
```diff
@@ -53,11 +53,12 @@
 				markerHeight="20"
 				markerUnits="userSpaceOnUse"
 				xoverflow="visible"
-				><path
+			>
```
Comment (echl) [inline packages/client/hmi-client/src/components/workflow/tera-simulation-workflow.vue:56]: This formatting happened automatically on merge

File: packages/client/hmi-client/src/components/workflow/tera-workflow-node.vue [hunk 3]
```diff
@@ -2,19 +2,13 @@
 	<main :style="nodeStyle" ref="workflowNode">
 		<header>
 			<h5>{{ node.operationType }}</h5>
-			<Button
-				icon="pi pi-ellipsis-v"
-				class="p-button-icon-only p-button-text p-button-rounded p-button-icon-only-small"
-			/>
+			<Button icon="pi pi-ellipsis-v"
```
Comment (echl) [inline packages/client/hmi-client/src/components/workflow/tera-workflow-node.vue:5]: This happened automatically on merge

**Comment by mwdchang** [comment]
Can you briefly describe the scope of this PR? 

It seems like it has nothing to do with the issue you want to close `Dataset detail pane` ... 
You modified the look and feel of the node, are we also address the need for a detail/drill-down pane here? or is that a separate upcoming PR?

File: packages/client/hmi-client/src/components/workflow/tera-dataset-node.vue [hunk 4]
```diff
@@ -76,4 +72,34 @@ section {
...
+	background-color: var(--gray-50);
+}
+
+:deep(.p-datatable .p-datatable-tbody > tr > td) {
+	border: none;
+}
+
+:deep(.p-datatable .p-datatable-thead > tr > th) {
+	border: none;
+}
```
Comment (YohannParis) [inline packages/client/hmi-client/src/components/workflow/tera-dataset-node.vue:104]: Why not updating the scss for p-datatable and add an extra classname that you can just apply, like `datatable-small` or something better?

Comment (echl) [reply packages/client/hmi-client/src/components/workflow/tera-dataset-node.vue:104]: Good idea

**Review by shawnyama — APPROVED** [review]

File: packages/client/hmi-client/src/components/workflow/tera-workflow-node.vue [hunk 5]
```diff
@@ -3,29 +3,48 @@
...
 		</header>
 		<ul class="inputs">
-			<li v-for="(input, index) in node.inputs" :key="index"
-				:class="input.status === WorkflowPortStatus.CONNECTED ? 'port-connected' : ''">
-				<div class="port" @click.stop="selectPort(input)" @mouseover="(event) => mouseoverPort(event)"
-					@focus="() => { }"></div>
+			<li
+				v-for="(input, index) in node.inputs"
+				:key="index"
+				:class="input.status === WorkflowPortStatus.CONNECTED ? 'port-connected' : ''"
```
Comment (YohannParis) [inline packages/client/hmi-client/src/components/workflow/tera-workflow-node.vue]:
```suggestion
				:class="{ 'port-connected': input.status === WorkflowPortStatus.CONNECTED }"
```

**Pull request merged**